charts/common/oidc-authservice/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v1.3"
```

Comment:
reference to 1.3?

Reply:
https://github.com/arrikto/oidc-authservice
Then it points to "This is a rewrite of the [ajmyyra/ambassador-auth-oidc](https://github.com/ajmyyra/ambassador-auth-oidc) project."
https://github.com/ajmyyra/ambassador-auth-oidc
https://github.com/ajmyyra/ambassador-auth-oidc/commit/a07b7bdf1e96aebf4d5b661f97b3fd35bda7f065

Reply:
strange, there are no tags or releases on this repo. for this, lets make it same as chart version

Reply:
done